{{cookiecutter.repostory_name}}/deploy.sh

```diff
@@ -2,10 +2,18 @@
 PRIMARY_IMAGE_HASH=$(docker-compose build app | tee /dev/tty \
     | sed '/^$/d' | grep "AS secondary-image" -B1 | head -1 | awk '{print $2}')
 docker-compose build  # build the rest of images
-docker-compose down -v --remove-orphans
-docker-compose up -d
+
+docker-compose restart app \
+{% if cookiecutter.use_celery == "y" -%}
+    celery-worker celery-beat \
+{%- endif %}
+{% if cookiecutter.use_flower == "y" -%}
+    celery-flower
+{%- endif %}
+
 docker-compose exec app python manage.py migrate
-docker-compose exec app python manage.py collectstatic
+docker-compose exec nginx nginx -s reload
```

Comment:
I doubt there are many of changes to the `nginx` service declaration in `docker-compose.yml` itself. I believe that most of the changes will be done to the nginx config so reloading it without the process downtime might be much better as it won't terminate connections established between clients and nginx

Reply:
sure, but please add a comment